src/oc-id/app/views/application/_analytics.html.erb

```diff
@@ -1,4 +1,3 @@
 <% if onetrust_enabled? %>
-  <!-- Onetrust Script -->
-  <script async src="https://cdn.cookielaw.org/consent/e231efa5-3ed9-4b92-96bc-f4c0872ca486/otSDKStub.js"  type="text/javascript" charset="UTF-8" data-domain-script="e231efa5-3ed9-4b92-96bc-f4c0872ca486"></script>
+  <%= javascript_include_tag "https://cdn.cookielaw.org/consent/#{ENV['ONETRUST_ID']}/otSDKStub.js", charset: "UTF-8", "data-domain-script": "#{ENV['ONETRUST_ID']}" %>
```

Comment:
@ayush-billore  check if ENV['ONETRUST_ID'] is not nil ?

Reply:
I have made this change @saghoshprogress

Reply:
is this working if ENV['ONETRUST_ID'] = '' ?

Reply:
I have made the change @saghoshprogress